trunk-recorder/uploaders/call_uploader.cc

```diff
@@ -207,23 +462,26 @@ void send_call(Call *call, System *sys, Config config) {
   boost::regex  ex("(http|https)://([^/ :]+):?([^/ ]*)(/?[^ #?]*)\\x3f?([^ #]*)#?([^ ]*)");
   boost::cmatch what;
 
-  if (regex_match(config.upload_server.c_str(), what, ex))
-  {
+  strcpy(call_info->filename,  call->get_filename());
```

Comment:
Moved this up so we'd still set these fields if we're only uploading to Broadcastify.